### PR TITLE
feat: adds scss preprocessor

### LINKED ISSUE #29: chore: add css preprocessor

### PR DESCRIPTION
**Pull request by dviramontes**
connects #29 
closes: https://github.com/codefordenver/members/issues/29

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,10 +1,7 @@
 .App-header {
   background-color: #e24e54;
   padding: 20px;
-  color: white;
-}
+  color: white; }
```
Comment (wdoug) [inline src/App.css]: U no like newlines?

Comment (dviramontes) [reply src/App.css]: thats the generated file ^

Comment (wdoug) [reply src/App.css]: Let's remove this from the repo and gitignore it

**Comment by wdoug** [comment]
:shipit:

**Pull request merged**